pom.xml

```diff
@@ -32,8 +32,8 @@ limitations under the License.
   <url>https://github.com/jenkinsci/flaky-test-handler-plugin</url>
 
   <properties>
-    <revision>1.2.4</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>1.3</revision>
```

Comment:
you've missed updating line 30, it should be either `${changelist}` or `${revision}.${changelist}`

see https://www.jenkins.io/doc/developer/publishing/releasing-cd/#update-maven-pom-and-config